index.js

```diff
@@ -238,12 +238,25 @@ class GithubScm extends Scm {
             this.octokitConfig.baseUrl = `${this.config.gheProtocol}://${this.config.gheHost}/api/v3`;
         }
 
+        let breakerOption = this.config.fusebox.breaker;
+
+        if (breakerOption) {
+            breakerOption.errorFn = function(err) {
+                return !(err.statusCode >= 400 && err.statusCode < 500);
```

Comment:
Is it possible that the `statusCode` is `undefined`? If so, you should separate the cases.

Reply:
I'll check err.statusCode just to be sure.